components/patient-data-sharing/push-ui/src/main/resources/PhenoTips/PushPatient.xml

```diff
@@ -414,25 +414,25 @@
         method: _this._container.method,
         parameters : {'do':'getuser', 'serverid': this._selectedServer },
         onCreate : function() {
-          console.log("PUSH: get username - start");
+          console &amp;&amp; console.log("PUSH: get username - start");
           _this._disableInputs();
         },
         onSuccess : function (response) {
-          //console.log("PUSH: Got user last used for the server (JSON: " + stringifyObject(response.responseJSON) + ")");
+          //console &amp;&amp; console.log("PUSH: Got user last used for the server (JSON: " + stringifyObject(response.responseJSON) + ")");
           if (response.responseJSON.remoteUserName) {
             _this._lastApprovedUser = response.responseJSON.remoteUserName;
           }
         },
         onFailure : function(response) {
-          console.log("PUSH: couldn't not get the user last used for this server");
+          console &amp;&amp; console.error("PUSH: couldn't not get the user last used for this server");
```

Comment:
Actually it's not expected behavior, I think `response.responseJSON.remoteUserName` is missing when the user is not authenticated yet.